openesdh-repo/src/main/java/dk/openesdh/repo/webscripts/files/OeFilesWebScript.java

```diff
@@ -115,6 +123,32 @@ public void addFileToCase(
         filesService.addToCase(caseId, file, title, docType, docCategory, description);
     }
 
+    @Uri(value = "/api/openesdh/files/upload/tmp", multipartProcessing = true, method = HttpMethod.POST, defaultFormat = WebScriptUtils.JSON)
```

Comment:
My intension of "OeFilesWebScript" was for user's files only. I sugest to move those actions to own TempFilesWebScript or someting similar